shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -53,10 +53,25 @@
     int UNKNOWN = 4;
   }
 
+  // Must match the PointerSignalKind enum in pointer.dart.
+  @IntDef({
+      PointerSignalKind.NONE,
+      PointerSignalKind.SCROLL,
+      PointerSignalKind.UNKNOWN
+  })
+  private @interface PointerSignalKind {
+    int NONE = 0;
+    int SCROLL = 1;
+    int UNKNOWN = 2;
+  }
+
   // Must match the unpacking code in hooks.dart.
   // TODO(mattcarroll): Update with additional fields for scroll wheel support
```

Comment:
Looks like this comment is obsolete now.

Reply:
Removed the comment